### DISCUSSION TITLE
How to use the projects?

### DISCUSSION THREAD

**Discussion by GDFan404** [pleonex/xdelta-sharp#15] [General]
i opened both xdeltasharp and create dump and both of them open and then close automaticly

**Comment by pleonex** [comment]
The focus of this project is to provide a programming library for .NET languages. The Visual Studio solution `Pleosoft.XdeltaSharp.sln` contains different type of C# projects:

- `PleOps.XdeltaSharp`: .NET library to decode VCDIFF files (xdelta). This can be integrated in other end-user frontend projects like a console or UI app.
- `PleOps.XdeltaSharp.Cli`: this is an example of a console app that decode a patch into a target file. It requires 3 arguments: `originalFile patchFile outputFile`

does it resolve your problem?

**Reply by GDFan404** [reply]
oh i thought i like decompiles the images for the xdelta its something else thanks though
